operator/tests/e2e-with-flags/nodepools/02-scale-down-first.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: nodepool-cluster
+spec:
+  image: "localhost/redpanda"
+  version: "dev"
+  nodePools:
+    - name: nodepool1
```

Comment:
Why there are 2 files `02-remove-first.yaml` and `02-scale-down-first.yaml`? I think the one should be removed.

Reply:
good catch. fixed